src/triage/cli.py

```diff
@@ -116,7 +120,7 @@ def __init__(self, parser):
         )
 
     def __call__(self, args):
-        self.root.setup  # Loading configuration (if exists)
+        self.root.setup()  # Loading configuration (if exists)
```

Comment:
This is really why I opened this branch. Just looking at these references, it appears that we would never invoke the setup at all, now that it's a proper method, (as opposed to a property).